src/main/java/com/github/KluevJakov/StoreClosure/service/ClosureService.java

```
@@ -0,0 +1,62 @@
+package com.github.KluevJakov.StoreClosure.service;
+
+import com.github.KluevJakov.StoreClosure.entity.Changelog;
+import com.github.KluevJakov.StoreClosure.entity.Closure;
+import com.github.KluevJakov.StoreClosure.repository.ChangelogRepository;
+import com.github.KluevJakov.StoreClosure.repository.ClosureRepository;
+import com.github.KluevJakov.StoreClosure.repository.StoreRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
+@Service
+public class ClosureService {
+    @Autowired
+    private ClosureRepository closureRepository;
+    @Autowired
+    private StoreRepository storeRepository;
+    @Autowired
+    private ChangelogRepository changelogRepository;
+
+    public boolean createClosureIfPossible(Long storeId, String text, Date startDate, Date endDate, String closureType) {
```

rename to createClosure() - after refactoring controller this method will receive Closure Dto and do not return boolean - instead return created Closure in database or nothing. Instead of returning false - throw exception